CMakeLists.txt

```diff
@@ -59,3 +59,11 @@ enable_testing()
 
 add_subdirectory(tests)
 
+#STARTUP MEX-TOOLBOX
+# The following line is to properly configure the installation script of the Mex-toolbox
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/startup_Mex.m.in ${CMAKE_BINARY_DIR}/startup_Mex.m)
+# Install configuration files
+install(FILES ${CMAKE_BINARY_DIR}/startup_Mex.m DESTINATION ${CMAKE_INSTALL_PREFIX}/mex)
```

Comment:
You can avoid to add explictly `${CMAKE_BINARY_DIR}` in the files because that is the binary in which CMake is currently working, and to add `${CMAKE_INSTALL_PREFIX}` from the DESTINATION argument due to how the [`install`](https://cmake.org/cmake/help/v3.0/command/install.html) CMake function works.